fhir/resources/core/fhirabstractmodel.py

```diff
@@ -9,13 +9,13 @@
 from enum import Enum
 from functools import lru_cache
 
+from pydantic import ValidationError
 from pydantic.v1 import BaseModel, Extra, Field
 from pydantic.v1.class_validators import ROOT_VALIDATOR_CONFIG_KEY, root_validator
-from pydantic.v1.error_wrappers import ErrorWrapper, ValidationError
-from pydantic.v1.errors import ConfigError, PydanticValueError
 from pydantic.v1.fields import ModelField
 from pydantic.v1.parse import Protocol
 from pydantic.v1.utils import ROOT_KEY, sequence_like
+from pydantic_core import InitErrorDetails
```

Comment:
`ValidationError`s now take a list of `InitErrorDetails` we should discuss how best to use this